contracts/drafts/ERC1046/ERC20Metadata.sol

```diff
@@ -7,18 +7,18 @@ import "../../token/ERC20/IERC20.sol";
  * @dev See https://eips.ethereum.org/EIPS/eip-1046
  * @dev tokenURI must respond with a URI that implements https://eips.ethereum.org/EIPS/eip-1047
  */
-contract ERC20TokenMetadata is IERC20 {
-    function tokenURI() external view returns (string memory);
-}
-
-contract ERC20WithMetadata is ERC20TokenMetadata {
```

Comment:
I think these renamings should be reflected in "breaking changes" in the changelog.